tests/conftest.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+from x12.config import X12Config
+
+
+@pytest.fixture
+def simple_270_with_new_lines():
+    return "\n".join(
+        [
+            "ISA*03*9876543210*01*9876543210*30*000000005      *30*12345          *131031*1147*^*00501*000000907*1*T*:~",
+            "GS*HS*000000005*54321*20131031*1147*1*X*005010X279A1~",
+            "ST*270*0001*005010X279A1~",
+            "BHT*0022*13*10001234*20131031*1147~",
+            "HL*1**20*1~",
+            "NM1*PR*2*PAYER C*****PI*12345~",
+            "HL*2*1*21*1~",
+            "NM1*1P*1*DOE*JOHN****XX*1467857193~",
+            "REF*4A*000111222~"
+            "N3*123 MAIN ST.*SUITE 42~"
+            "N4*SAN MATEO*CA*94401~"
+            "HL*3*2*22*0~"
+            "TRN*1*930000000000*9800000004*PD~",
+            "NM1*IL*1*DOE*JOHN****MI*00000000001~",
+            "REF*6P*0123456789~",
+            "DMG*D8*19700101~",
+            "DTP*291*D8*20131031~",
+            "EQ*1~",
+            "SE*17*0001~",
+            "GE*1*1~",
+            "IEA*1*000000907~",
+        ]
+    )
+
+
+@pytest.fixture
+def simple_270_one_line(simple_270_with_new_lines):
+    return simple_270_with_new_lines.replace("\n", "")
+
+
+@pytest.fixture
+def large_x12_message():
+    """
+    Generates a large x12 message by repeating a transaction set a fixed number of times.
+    :return: large x12 message
+    """
+    x12_message: str = "\n".join(
+        [
+            "ISA*03*9876543210*01*9876543210*30*000000005      *30*12345          *131031*1147*^*00501*000000907*1*T*:~",
+            "GS*HS*000000005*54321*20131031*1147*1*X*005010X279A1~",
+        ]
+    )
+
+    x12_message += "\n"
+    # will create 5000 transaction sets with 17 segments each
+    for i in range(1, 5_001, 1):
```

Comment:
Is 5_001 a typo?

Reply:
@ccorley Python allows you to use the underscore as a separator in numeric literals. I've started using them to make things a bit easier to read!

Reply:
Got it - it's just a visual separator that's parsed as if it were not there.